shared/test/frontend_tester.py

```diff
@@ -17,14 +17,59 @@
 # need to consider whether this code wants to be shared with WebDriver tests
 # for other browsers, though.
 #
+
+gCommonDir = None
```

Comment:
Adding a comment explaining where this needs to be overridden and why would be nice.